### PR TITLE
make it possible to make/publish appeals

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR makes it possible to make an draft appeal and thus "publishing" it.
One thing i realized: maybe we are using the wrong language here? i would not understand the difference between "making a draft" and then "making the appeal"? At least in German it does not work that way. But we also have no dedicated word like "appeal" or a phrase like "making an appeal"

For me `publish` is much clearer (also in German). Because in my mind "i already made the appeal" (i created an appeal thingy and can see it there in the UI)

bildschirmfoto 2015-12-04 um 14 36 30

### CODE REVIEW

**Comment by rhyslbw** [comment]
So in English, "Draft an appeal" -> "Make the appeal"

**Comment by rhyslbw** [comment]
How would you describe the action the org admin is taking here in say German translated to English :-)

**Comment by DominikGuzei** [comment]
I have to talk with the domain experts about that. But my feeling is, that we don't have special words for these things. It's more like "make it public" vs. "this is a draft"

**Comment by rhyslbw** [comment]
but _what_ is being made public?

**Comment by DominikGuzei** [comment]
the appeal – there is no clear word for that in German.
there are terms similar to "donation request" but that's not "officially" the term.

**Comment by DominikGuzei** [comment]
I used the term "Spendenanfrage" now in German, which is similar to "donation request".
And instead of "make" i use "veröffentlichen" which means "make public"

File: packages/app/lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -33,5 +33,10 @@
   "street": "Street",
   "zip": "Zip",
   "city": "City",
-  "open_hours": "Open hours"
+  "open_hours": "Open hours",
+  "appeals": {
+    "draft": "<strong>Draft</strong>: This appeal is not yet made.",
```
Comment (rhyslbw) [inline packages/app/lib/i18n/en.i18n.json]: "This appeal is not yet public." would be the best way to phrase this.

Comment (DominikGuzei) [inline packages/app/lib/i18n/en.i18n.json]: Ok thanks! Please take over there – as you're the expert for English :wink: 
im already on the next PR

**Pull request merged**